logger/logger.go

```diff
@@ -42,7 +42,7 @@ func InitLogger() {
 		}
 		log.AddHook(hook)
 		log.SetFormatter(&log.JSONFormatter{
-			TimestampFormat: time.Now().Format("2006-01-02T15:04:05.999Z"),
+			TimestampFormat: "2006-01-02T15:04:05.999Z",
```

Comment:
Can we do....

```suggestion
			TimestampFormat: time.RFC3399,
```

?

Reply:
Or RFC3339Nano

Reply:
Updated to use `time.RFC3339Nano`.